new-website/src/pages/partner/locations/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react"
+import queryString from 'query-string'
+import { EditPage } from '../../../components/location/edit-page'
+import MuiAlert from "@material-ui/lab/Alert"
+import { useIntl } from "gatsby-plugin-intl"
+import { Protected } from "../../../components/protected"
+import Layout from "../../../components/layout"
+import { Snackbar } from "@material-ui/core"
+import { ErrorBoundary } from "../../../components/error/error-boundary"
+
+function Alert({ open, severity, message, onClose }) {
+    return (
+        <Snackbar
+            anchorOrigin={{ vertical: "bottom", horizontal: "center" }}
+            autoHideDuration={6000}
+            open={open}
+            onClose={onClose}
+        >
+            <MuiAlert elevation={6} variant="filled" severity={severity}>
+                {message}
+            </MuiAlert>
+        </Snackbar>
+    )
+}
+
+const EditLocations = ({ location }) => {
+    const [alert, setAlert] = useState({ message: null, severity: "error", open: false })
+    const closeAlert = () => setAlert({ ...alert, open: false })
+
+    const { formatMessage } = useIntl()
+    const onError = ({ code }) => {
+        setAlert({
+            open: true,
+            message: formatMessage({ id: code }),
+            severity: "error",
+        })
+    }
+
+    const editParam = queryString.parse(location?.search)?.edit || 'new'
+    return (
+        <ErrorBoundary>
```

Comment:
This Error Boundary should not be needed unless we have an error in the base layout?

Reply:
Fixed with https://github.com/ImmunHelden/ImmunHelden.de/pull/155/commits/f72c183134a738dd96f193fbd2296b8771c8271d